main.js

```diff
@@ -0,0 +1,234 @@
+//Tic-Tac-Toe 2.20.20
+
+//start game
+//->user clicks on "start"
+//--start button grayed out --status area says "Player X's turn"
+
+// document.getElementById('start').on('click', ':button', function (e) {
+//     let btn = $(e.target);
+//     btn.attr("disabled", "disabled"); // disable button
+
+// });
+
+
+//current player
+//->user selects cell
+//--board redraws --x (or O) appears in that cell
+//--turn ends --current player goes to opposite (x or o)
+// onclick document.getElementById()
+//     ->sends cell value to winCheck; 
+//     ->replaces empty cell with x or o
+//     ->playerTurn ++
+
+//user select cell
+//->user picks an already-filled cell
+//--game says "Pleaase select an empty cell" 
+//--current player does _not_ change
```

Comment:
You should delete these comments out as you complete them to help keep your file clean.